packages/compass-sidebar/src/components/sidebar/sidebar.jsx

```diff
@@ -6,11 +6,11 @@ import ReactTooltip from 'react-tooltip';
 import { AutoSizer, List } from 'react-virtualized';
 import { globalAppRegistryEmit } from '@mongodb-js/mongodb-redux-common/app-registry';
 import { Resizable } from 're-resizable';
```

Comment:
Something to consider for the future maybe. Would it make sense to move this library usage in compass-components also? Seems like we are repeating a lot of pretty similar code between the two otherwise 🤔

Reply:
Good thinking, would be worth generalizing when/if we add more occurrences of splitters in the app 👍